### ISSUE TITLE
powstream errors

### ISSUE THREAD

**Issue by bltierney** [perfsonar/pscheduler#302]
Periodically get this error. 

This is benign, so perhaps we ignore it, and not archive it?

Test Failures
<- [latency] powstream exited with error, will attempt restart: powstream[29527]: open(/var/lib/pscheduler/tool/powstream/418ba7f1-d4b2-4e5b-9452-f6268516846d/15872854643823794071_15872854897179377872.owp.i): No such file or directory
-> [latency] powstream exited with error, will attempt restart: powstream[29379]: open(/var/lib/pscheduler/tool/powstream/f48829c7-9872-48fc-8c71-8a3766ad157f/15872854640052928646_15872854901221957965.owp.i): No such file or directory

**Comment by arlake228** [comment]
Looked at this a bit today and I understand what's happening. Basically these crop-up when a yum update hits and pscheduler-runner gets restarted. You can very eailt correlate the dots with yum.log entries. Its also relatively easy to recreate by restart pscheduler-runner. They arise because the old powstream/run processes don't die right away so you have the old ones and new ones running in parallel for up to 60 seconds. When the old ones finally do die, they wipe their data directory, which essentially sweeps the rug out from under the new processes and they report this error. The new processes then rebuild the directory and function normally. 

I'm looking at ways to make the old processes die sooner so we don't get this overlap. Also thinking about randomizing the name of the data dir a bit more, but also don't want to sacrifice some of the cleanup advantages its gives us, so need to think on that some more.

**Comment by mfeit-internet2** [comment]
I'd recommend having the `run` method create a unique temporary directory with `tempfile.mkdtemp()` and let powstream work in that.  That leaves no chance that any instance of powstream can interfere with any other.

**Comment by bltierney** [comment]
If this only happens when the rpm gets updated, I'd consider this pretty low priority to fix....
